lib/happo/public/happo-runner.js

```diff
@@ -199,6 +199,11 @@ window.happo = {
       this.getFullRectRecursive(node.children[i], box);
     }
 
+    // Cut off things that render off the screen to the top or left, since those
+    // won't be in the screenshot file that we then crop from.
+    box.left = Math.max(box.left, 0);
+    box.top = Math.max(box.top, 0);
```

Comment:
And, we don't need to do this for the right or bottom because the screenshot will always include that, even if it goes off the page, right?

I wonder if this is actually a bug in webdriver...